<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NTB-257" failures="1" tests="2" name="Surefire test" time="13.797" errors="0" timestamp="2022-11-24T15:14:10 CET">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.052"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.518">
    <failure type="java.lang.AssertionError" message="did not expect [JovanK] but found [JovanK]">
      <![CDATA[java.lang.AssertionError: did not expect [JovanK] but found [JovanK]
at Academy.StepDefinitions.StepDefinition.verify_that_the_users_is_not_as_expected(StepDefinition.java:145)
at âœ½.Verify that the user's JovanK is not as expected(file:///C:/Users/joko2909/SeleniumTraining/Toma14112022/src/test/java/Academy/Cucumber/ErrorValidations.feature:24)
... Removed 5 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Surefire test -->
